checks/system/slurm/slurm.py

```diff
@@ -209,29 +209,31 @@ def set_memory_limit(self):
 
 @rfm.simple_test
 class MemoryOverconsumptionMpiCheck(SlurmCompiledBaseCheck):
-    maintainers = ['JG']
-    valid_systems += ['eiger:mc', 'pilatus:mc', 'hohgant:gpu']
+    maintainers = ['@jgphpc', '@ekouts']
+    valid_systems += ['*']
     time_limit = '5m'
+    build_system = 'SingleSource'
     sourcepath = 'eatmemory_mpi.c'
     tags.add('mem')
-    executable_opts = ['100%']
+
+    @run_before('compile')
+    def set_skip(self):
+        # TODO: use env. features: if 'alps' then yes
```

Comment:
@ekouts how would you read the features from the current_env ?

Reply:
`self.current_environ.features` and `self.current_environ.extras` but it has to be after the setup phase